clients/js/src/defaultGuards/default.ts

```diff
@@ -92,6 +95,7 @@ export type DefaultGuardSetArgs = GuardSetArgs & {
   freezeTokenPayment: Option<FreezeTokenPaymentArgs>;
   programGate: Option<ProgramGateArgs>;
   allocation: Option<AllocationArgs>;
+  token2022Payment: Option<Token2022PaymentMintArgs>;
```

Comment:
That should be the guard's args not their mint args.

```suggestion
  token2022Payment: Option<Token2022PaymentArgs>;
```